lib/src/_shared/PickerBase.jsx

```diff
@@ -76,6 +76,10 @@ export default class PickerBase extends PureComponent {
   }
 
   handleTextFieldChange = (date) => {
-    this.setState({ date }, this.handleAccept);
+    if (date === null) {
```

Comment:
Do we need to duplicate `onChange(null)` logic? It's already defined in separate `handleClear` method

Reply:
You are right, I changed it.